cmd/lib/ci_matrix.rb

```diff
@@ -85,8 +85,13 @@ def self.architectures(cask_content:)
     end
   end
 
-  def self.random_runner(avalible_runners = INTEL_RUNNERS)
-    avalible_runners.max_by { |(_, weight)| rand ** (1.0 / weight) }
+  def self.random_runner(available_runners = INTEL_RUNNERS)
+    available_runners.max_by { |(_, weight)| rand ** (1.0 / weight) }
+                    .first
+  end
+
+  def self.style_runner(available_runners = ARM_RUNNERS)
+    available_runners.min_by { |(_, weight)| weight }
```

Comment:
```suggestion
    available_runners.max_by { |(_, weight)| weight }
```

when there's multiple we want the one with the highest weight

though tbh you can just probably use the existing implementation and was the default `INTEL_RUNNERS` for `ARM_RUNNERS` given the style check is the only place the default value is used.

Reply:
No idea what I was doing there. Thanks.